### PR TITLE
144 add a wizard for new users

### LINKED ISSUE #144: ✨ Add a wizard for new users
### Contact Details

_No response_

### What is it you truly desire

Add a wizard for new users

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by orlowdev**
Closes #144

### CODE REVIEW

File: src/client/activity-bar/index.tsx [hunk 1]
```diff
@@ -38,34 +38,34 @@ export default function ActivityBar() {
...
-      action: (_, { dispatch }) => dispatch(selectActivity("achievements")),
-    },
-    {
-      title: "@extensions/open-activity",
-      icon: "BsPuzzle",
-      accelerator: "ctrl+alt+e",
-      showInCommandPalette: true,
-      action: (_, { dispatch }) => dispatch(selectActivity("extensions")),
-    },
+    // {
```
Comment (aleksandr-konovalov) [inline src/client/activity-bar/index.tsx:41]: Need a TODO, if you want to leave comments

File: src/client/common/components/button.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  disabled,
+}: Props) => {
+  const buttonAppearanceClass = Either.fromBoolean(!!disabled).fold(
+    () =>
+      "bg-gradient-to-r from-sky-200 dark:from-cyan-600 via-teal-200 dark:via-teal-600 to-emerald-200 dark:to-emerald-600 active-ring",
+    () =>
+      "bg-gradient-to-r from-slate-300 via-zinc-300 to-stone-300 dark:from-slate-900 dark:via-zinc-900 dark:to-stone-900"
+  )
+
+  const buttonClass = `${buttonAppearanceClass} ${className}`
```
Comment (aleksandr-konovalov) [inline src/client/common/components/button.tsx:126]: I think we need "classnames" to concat classes

File: src/client/create-modal/index.tsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ export default function CreateModal() {
                 if (e.key === "Escape") {
                   hideModal()
                 } else if (e.key === "Enter") {
-                  dispatch(create(directory.path + separator + newName))
+                  dispatch(create({ path: directory.path + separator + newName }))
```
Comment (aleksandr-konovalov) [inline src/client/create-modal/index.tsx:64]: Please, use template literals

File: src/main/index.ts [hunk 4]
```diff
@@ -88,7 +88,7 @@ const createWindow = async () => {
   // and load the index.html of the app.
   mainWindow.loadURL(MAIN_WINDOW_WEBPACK_ENTRY)
 
-  mainWindow.webContents.toggleDevTools()
+  // mainWindow.webContents.toggleDevTools()
```
Comment (aleksandr-konovalov) [inline src/main/index.ts:91]: :(

**Review by aleksandr-konovalov — APPROVED** [review]

File: src/client/create-modal/index.tsx (hunk 3 above)
Comment (orlowdev) [reply src/client/create-modal/index.tsx:64]: Makes sense, this is a quick fix. Thanks!

File: src/main/index.ts (hunk 4 above)
Comment (orlowdev) [reply src/main/index.ts:91]: You can open dev tools with a keyboard combo if you need it. It still works, it just doesn't open at the app launch.

File: src/client/activity-bar/index.tsx (hunk 1 above)
Comment (orlowdev) [reply src/client/activity-bar/index.tsx:41]: It was slightly above, moved it to the first line of the comment

File: src/client/common/components/button.tsx (hunk 2 above)
Comment (orlowdev) [reply src/client/common/components/button.tsx:126]: True, just a sec

**Review by aleksandr-konovalov — APPROVED** [review]

**Pull request merged**